packages/devextreme/js/ui/validation_engine.js

```diff
@@ -27,6 +27,11 @@ const STATUS = {
     pending: 'pending'
 };
 
+const validGroupNames = [
```

Comment:
[naming]
I didn't get what do you mean when you say "validGroup". It can be not valid as well (containing failed rules).
Also we usually use UPPER_SNAKE_CASE for constants.
```suggestion
const NON_DELETABLE_GROUP_NAMES = [
```

P.s I write this not just as a feedback for you to not repeat it in the future.